.circleci/config.yml

```diff
@@ -1,14 +1,194 @@
+            echo 'export GRAPHQL_SERVER_IMAGE="litmusportal-server"' >> workspace/env-vars
+            echo 'export AUTHENTICATION_SERVER_IMAGE="litmusportal-auth-server"' >> workspace/env-vars
+            echo 'export FRONTEND_IMAGE="litmusportal-frontend"' >> workspace/env-vars
+            echo 'export SUBSCRIBER_IMAGE="litmusportal-subscriber"' >> workspace/env-vars
+            echo 'export SELF_DEPLOYER_IMAGE="litmusportal-self-deployer"' >> workspace/env-vars
+            echo 'export IMGTAG="ci"' >> workspace/env-vars
+            cat workspace/env-vars >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Build graphql server docker image
+          command: docker build . -f build/Dockerfile -t ${REPONAME}/${GRAPHQL_SERVER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/graphql-server
+      - run:
+          name: Save graphql server docker image
+          command: docker save -o workspace/graphql-server.tar ${REPONAME}/${GRAPHQL_SERVER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build auth server docker image
+          command: docker images && docker build . -f Dockerfile -t ${REPONAME}/${AUTHENTICATION_SERVER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/auth
+      - run:
+          name: Save auth server docker image
+          command: docker save -o workspace/auth-server.tar ${REPONAME}/${AUTHENTICATION_SERVER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build subscriber docker image
+          command: docker build . -f build/Dockerfile -t ${REPONAME}/${SUBSCRIBER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/subscriber
+      - run:
+          name: Save subscriber docker image
+          command: docker save -o workspace/subscriber.tar ${REPONAME}/${SUBSCRIBER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build self-deployer docker image
+          command: docker build . -f build/Dockerfile -t ${REPONAME}/${SELF_DEPLOYER_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/backend/tools/self-deployer
+      - run:
+          name: Save self-deployer docker image
+          command: docker save -o workspace/self-deployer.tar ${REPONAME}/${SELF_DEPLOYER_IMAGE}:${IMGTAG}
+      - run:
+          name: Build frontend docker image
+          command: docker build . -f Dockerfile -t ${REPONAME}/${FRONTEND_IMAGE}:${IMGTAG}
+          working_directory: ~/project/litmus-portal/frontend
+      - run:
+          name: Save frontend docker image
+          command: docker save -o workspace/frontend.tar ${REPONAME}/${FRONTEND_IMAGE}:${IMGTAG}
+      - persist_to_workspace:
+          root: workspace
+          paths:
+            - ${GRAPHQL_SERVER_IMAGE}.tar
+            - ${SUBSCRIBER_IMAGE}.tar
+            - ${SELF_DEPLOYER_IMAGE}.tar
+            - ${AUTHENTICATION_SERVER_IMAGE}.tar
+            - ${FRONTEND_IMAGE}.tar
```

Comment:
is it not required to persist the envs also?

Reply:
Ohh, forgot to add.